resources/js/views/settings/Application.vue

```diff
@@ -166,139 +166,212 @@
         </b-form-group>
 
         <!--Room limit settings-->
-        <b-form-group
-          label-class="font-weight-bold"
-          class="mb-4"
-          label-for="application-room-limit-input"
-          :description="$t('settings.application.roomLimit.description')"
-          :state='fieldState("room_limit")'
-          :label="$t('settings.application.roomLimit.title')"
-        >
-          <b-form-radio-group
-            class='mb-2'
-            id="application-room-limit-radio-group"
-            v-model='roomLimitMode'
-            :options='roomLimitModeOptions'
-            :disabled='isBusy || viewOnly || !loaded'
-            required
-            :state='fieldState("room_limit")'
-            @change="roomLimitModeChanged"
-            stacked
-          ></b-form-radio-group>
-
-          <b-form-input
-            id='application-room-limit-input'
-            type='number'
-            :state='fieldState("room_limit")'
-            v-model='settings.room_limit'
-            min='0'
-            max="100"
-            required
-            :disabled='isBusy || viewOnly || !loaded'
-            v-if="roomLimitMode === 'custom'">
-          </b-form-input>
-
-          <template slot='invalid-feedback'>
-            <div v-html="fieldError('room_limit')"></div>
-          </template>
-        </b-form-group>
-
-        <!--Pagination page size settings-->
-        <b-form-group
-          label-class="font-weight-bold"
-          class="mb-4"
-          label-for="application-pagination-page-size-input"
-          :description="$t('settings.application.paginationPageSize.description')"
-          :state='fieldState("pagination_page_size")'
-          :label="$t('settings.application.paginationPageSize.title')"
-        >
-          <b-form-input id="application-pagination-page-size-input"
-                        v-model="settings.pagination_page_size"
-                        type="number"
-                        :disabled="isBusy || viewOnly || !loaded"
-                        min="1"
-                        max="100"
-                        required
-                        :state='fieldState("pagination_page_size")'
-          >
-          </b-form-input>
-
-          <template slot='invalid-feedback'>
-            <div v-html="fieldError('pagination_page_size')"></div>
-          </template>
-        </b-form-group>
+        <b-row cols='12'>
+          <b-col md='6'>
+            <b-form-group
```

Comment:
I would prefer a heading like all the other settings 'block's:
![image](https://user-images.githubusercontent.com/4281791/110973858-19538300-835e-11eb-80f8-4f32eb03f10a.png)